src/cat.js

```diff
@@ -32,12 +32,9 @@ function _cat(options, files) {
     if (!fs.existsSync(file))
       common.error('no such file or directory: ' + file);
 
-    cat += fs.readFileSync(file, 'utf8') + '\n';
+    cat += fs.readFileSync(file, 'utf8');
   });
 
-  if (cat[cat.length-1] === '\n')
```

Comment:
Why is this there? Wouldn't it actually be incorrect.

Example:

```
$ printf "foo\n" > foo # printf, because it properly handles `\n`.
$ printf "bar\n" > bar
$ cat foo bar
foo
bar
$ # There was a newline above this prompt.
```

With this line, there would be no newline at the end.